webapp/channels/src/components/channel_header_dropdown/menu_items/view_pinned_posts/view_pinned_posts.tsx

```diff
@@ -19,33 +19,32 @@ type Props = {
     };
 }
 
-export default class ViewPinnedPosts extends React.PureComponent<Props> {
-    private handleClick = (e: React.MouseEvent) => {
+const ViewPinnedPosts = ({
+    channel,
+    hasPinnedPosts,
+    actions: {
+        closeRightHandSide,
+        showPinnedPosts,
+    },
+    show,
+}: Props) => {
+    const handleClick = useCallback((e: React.MouseEvent) => {
```

Comment:
nit
```suggestion
    const handleClick = useCallback((e: MouseEvent) => {
```

import from React